examples/wdio/multiremote/mocha.test.js

```diff
@@ -5,26 +5,32 @@ describe('multiremote example', () => {
 
     it('should login the browser', () => {
         const nameInput = $('.usernameInput')
-        nameInput.addValue('Browser A')
+        nameInput.addValue('Browser')
```

Comment:
The idea here is that one browser is logging in with "Browser A" and the other with "Browser B" let's keep this even though it wasn't quite right before.

Reply:
it is not about being generic. The use case for multibrowser is to simulate multiple "actors". So in this case e.g. player A and player B

Reply:
@christian-bromann If you want I can change the label back to 'Browser A', for me it's all the same.